examples/structured_data/movielens_recommendations_transformers.py

```diff
@@ -0,0 +1,606 @@
+"""
+Title: Recommendations with Transformer
```

Comment:
Proposal: "recommendation" + "system"

```suggestion
Title: A recommendation system with a transformer
```

The architecture in a nutshell is as follow: an embedding layer, a transformer layer, and an MLP. The transformer is what differentiates the "BST" model from the other one ("WDL")  https://arxiv.org/pdf/1905.06874.pdf.